sdlf-ddk-lightweight/data_lake/src/lambdas/sdlf_heavy_transform/routing/handler.py

```diff
@@ -42,6 +43,7 @@ def lambda_handler(event, context):
         org = event['org']
         app = event['app']
         env = event['env']
+        database_name = event['database_name']
```

Comment:
Just to clarify here - will db name be present in the event bridge event that comes from the bucket notification? What will put it in there?

Reply:
For StageB routing (on a scheduled basis) we set a constant RuleTargetInput and are now adding the database name as an additional parameter (can refer to `register_dataset()` function in file `data_lake/pipelines/standard_pipeline/standard_pipeline.py`) in order to catalog the new files within the glue job

Reply:
Gotcha, thanks.